app/models/consent.rb

```diff
@@ -190,6 +192,16 @@ def reasons_triage_needed
   end
 
   def self.from_consent_form!(consent_form, patient_session)
+    parent =
```

Comment:
Can you wrap this in a transaction, to prevent the creation of orphaned parent records in case the consent creation fails later?

Reply:
Yup – added in 4534c03c80036f35b57cfee0f9c6c0f3ed3f02c0